clusterloader2/testing/load/config.yaml

```diff
@@ -160,39 +160,28 @@ steps:
         objectTemplatePath: daemonset-priorityclass.yaml
 
 - name: Starting measurement for waiting for pods
-  measurements:
-  - Identifier: WaitForRunningDeployments
-    Method: WaitForControlledPodsRunning
-    Params:
-      action: start
+  Method: WaitForControlledPodsRunning
+  Configs:
```

Comment:
What is Configs?
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/api/types.go#L175

Reply:
configs is Instances. I took the config from the old conversation. sorry should have checked.